sql/core/src/main/scala/org/apache/spark/sql/jdbc/MySQLDialect.scala

```diff
@@ -45,7 +45,46 @@ private case class MySQLDialect() extends JdbcDialect with SQLConfHelper with No
   // See https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html
   private val supportedAggregateFunctions =
     Set("MAX", "MIN", "SUM", "COUNT", "AVG") ++ distinctUnsupportedAggregateFunctions
-  private val supportedFunctions = supportedAggregateFunctions ++ Set("DATE_ADD", "DATE_DIFF")
+
+  // See https://dev.mysql.com/doc/refman/8.4/en/built-in-function-reference.html
+  // The functions listed here have the same signature and similar semantics,
+  // and can be supported with existing mechanism.
+  private val supportedSQLFunctions = Set(
```

Comment:
I think we can split this set of functions according to Spark's classification of functions, for example, `ABS` belongs to `Mathematical Functions`, `COALESCE` to `Conditional Functions`, `LOWER` to `String Functions`, and so on.

Reply:
Sure, let me fix it.

Reply:
It is fixed now, thanks for the recommendation. PTAL.